__tests__/components/Table.spec.js

```diff
@@ -361,6 +361,172 @@ describe('Table component', () => {
 
       expect(cellsText).toEqual(dataText)
     })
+
+    it('should update the table when data is added', () => {
+      const field = 'Name'
+      const cols = [
+        {
+          field
+        }
+      ]
+
+      const data = [
+        {
+          Name: 'Caterpie',
+          description: 'lorem ipsum dolor sit amet'
+        },
+        {
+          Name: 'Blastoise',
+          description: 'consecteur adipiscing elit, sed do eiusmod'
+        }
+      ]
+
+      const component = mount(
+        <Table columns={cols} data={data} />
+      )
+
+      const cellsText = component.find('[data-display="table-body"] [data-display="table-cell"]')
+        .map(element => element.text())
+
+      const dataText = _.map(data, (field))
```

Comment:
I was initially confused, but I guess that we don't need the parentheses, it seems weird when compared with an actual mapping function.
Or maybe even drop the `field` var since we only use it a few places and IMO it would improve readability.